README.md

```diff
@@ -23,3 +23,4 @@ This is a repository full of implementations of the classic, "Hello, world!", in
 - Scala
 - Swift
 - TypeScript
+- Julia
```

Comment:
Could you put `Julia` in alphabetical order?

Reply:
done